src/libsystemd-network/ndisc-option.c

```diff
@@ -65,6 +65,13 @@ static sd_ndisc_option* ndisc_option_new(uint8_t type, size_t offset) {
         return p;
 }
 
+static void ndisc_raw_done(sd_ndisc_raw *raw) {
+        if (!raw)
+                return;
+
+        free(raw->bytes);
```

Comment:
raw is not freed, so better to use mfree here and set length to 0

Reply:
Our coding style is (though that's not strict)
- `xyz_done()`: can be called at most once per object, and the passed object must not be reused.
- `xyz_clear()`: can be called multiple times, and the passed object can be reused.
- `xyz_free()`: frees passed object, so the passed object cannot be reused.

In this case, the function is named as `_done()`, and the object will never reused, so that's not necessary.